### DISCUSSION TITLE
Corresponding versions information needed

### DISCUSSION THREAD

**Discussion by RSoodAMAT** [open-telemetry/opentelemetry-java#3247] [Q&A]
Needed the corresponding information related to open telemetry 

Open telemetry version : 1.2.0 corresponds to which version of otel collector and otel java agent. 
Also need information for open telemetry version 1.1.0 , what are the corresponding versions for above libraries.

**Comment by iNikem** [comment]
There is no relation between the releases of this repository and OpenTelemetry Collector.

Releases of OpenTelemetry Java instrumentation document the corresponding version of OpenTelemetry Java in their [release notes](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
